votes/views.py

```diff
@@ -124,6 +152,46 @@ def form_valid(self, form):
         return super().form_valid(form)
 
 
+class ResetVoteView(PermissionRequiredMixin, FormView):
+    permission_required = PERMS.votes.change_ticket
+    form_class = ResetVoteForm
+    template_name = "votes/reset_vote.html"
+    success_url = reverse_lazy('votes:admin')
+
+    def form_valid(self, form):
+        uuid = form['uuid']
```

Comment:
```suggestion
        uuid = form.cleaned_data['uuid']
```
as you told me to

Reply:
Fixed in 29c9c2f6ee25c8d60dade845bc555b3d53a2e7a6